app/assets/src/components/views/samples/SamplesView.jsx

```diff
@@ -191,21 +192,31 @@ class SamplesView extends React.Component {
         ? mapSidebarSelectedSampleIds
         : selectedSampleIds;
 
-    const log = () =>
-      logAnalyticsEvent("SamplesView_heatmap-icon_clicked", {
-        selectedSampleIds: targetSampleIds.length,
+    const heatmapOptions = [
```

Comment:
If `props.admin` is false, keep the `<a>` link as before, instead of showing a dropdown with just one option.

Reply:
We don't want to change the UI for non-admin users until we launch the feature.